css/at-rules/custom-media.json

```
@@ -0,0 +1,41 @@
+{
+  "css": {
+    "at-rules": {
+      "custom-media": {
+        "__compat": {
+          "description": "`@custom-media`",
+          "spec_url": "https://drafts.csswg.org/mediaqueries-5/#at-ruledef-custom-media",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "impl_url": "https://crbug.com/40781325"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "146",
```

Actually, it's not enabled by default:

```suggestion
              "version_added": "146",
              "flags": [
                {
                  "type": "preference",
                  "name": "layout.css.custom-media.enabled",
                  "value_to_set": "true"
                }
              ],
```